ide/notifications/src/org/netbeans/modules/notifications/Bundle.properties

```diff
@@ -21,6 +21,7 @@ OpenIDE-Module-Display-Category=Notifications
 OpenIDE-Module-Name=Notifications
 #module short description
 OpenIDE-Module-Short-Description=Notifications pup-ups and Notifications Center
+OpenIDE-Module-Long-Description=Provides IDE notifications pup-ups and Notifications Center
```

Comment:
is it pop-ups ?

Reply:
It is for show a description in plugin manager: 
![Screenshot from 2020-03-14 19-16-38](https://user-images.githubusercontent.com/4323228/76687939-c51da200-6628-11ea-9731-1fb03c83b541.png)

Before:
![Screenshot from 2020-03-14 19-17-11](https://user-images.githubusercontent.com/4323228/76687987-1594ff80-6629-11ea-9ef4-7f43aa0dbc2c.png)

Reply:
ok but the words were not good I guess pup up should be pop-up